.github/CODEOWNERS

```diff
@@ -1,3 +1 @@
-# Read more about CODEOWNERS here - https://github.com/blog/2392-introducing-code-owners
-
-*   @procore/team-api
+* @procore-oss/procore-ospo @procore-oss/procore-ruby-sdk
```

Comment:
please ignore: will be correct once we move repo to github.com/procore-oss github org